test/tests/utils/spline_interpolation/tests

```diff
@@ -1,8 +1,13 @@
 [Tests]
-  [./test]
+  [./regular_spline]
     type = 'Exodiff'
     input = 'spline_interpolation.i'
     exodiff = 'spline_interpolation_out.e'
   [../]
+  [./bicubic_spline]
+    type = 'Exodiff'
+    input = 'bicubic_spline_interpolation.i'
+    exodiff = 'bicubic_spline_interpolation_out.e'
+    custom_cmp = 'bicubic_spline_interpolation.cmp'
+  [../]
 []
-  
```

Comment:
Dang, I missed this on the original review and it's going to show up as a diff if people run the remove trailing whitespace script. Will you remove the extra whitespace on this line then I'll merge. Don't delete this line, just remove the space.